particula/dynamics/coagulation/transition_regime.py

```diff
@@ -30,6 +34,13 @@ def hard_sphere(
     Journal of Chemical Physics, 126(12).
     https://doi.org/10.1063/1.2713719
     """
+    if np.any(diffusive_knudsen < 0):
```

Comment:
probably not the best way to scale input validations, we should think about that going forward